stream-platform/src/main/java/io/camunda/zeebe/stream/impl/records/TypedRecordImpl.java

```diff
@@ -38,6 +38,7 @@ public void wrap(
     this.value = value;
   }
 
+  @JsonIgnore
```

Comment:
❓ Why was this necessary?

Reply:
Because otherwise it's serialized. See the other fields in this class.

Reply:
But why was it necessary NOW ?

Reply:
Because I added the method `getMetadata` that gets picked up by Jackson such that all metadata is serialized again. We don't want that so I also had to add `JsonIgnore` to prevent it.